code/datalab-app/src/api/dataaccess/vault/vault.spec.js

```diff
@@ -0,0 +1,132 @@
+import moxios from 'moxios';
+import logger from 'winston';
+import vault from './vault';
+import config from '../../config';
+
+jest.mock('winston');
+
+beforeEach(() => {
+  moxios.install();
+});
+
+afterEach(() => {
+  moxios.uninstall();
+  logger.clearMessages();
+});
+
+const secretPath = 'datalab/files/files1';
+const loginUrl = `${config.get('vaultApi')}/v1/auth/approle/login`;
+const secretUrl = `${config.get('vaultApi')}/v1/secret/`;
+
+test('that a token request calls the correct endpoint', () => {
```

Comment:
Rename these tests for consistency, for previous tests we have used `describe` and `it` terminology.